webviz_subsurface/containers/_inplace_volumes_onebyone.py

```diff
@@ -139,6 +140,55 @@ def add_webvizstore(self):
             ),
         ]
 
+    @property
+    def tour_steps(self):
+        return [
+            {
+                "id": self.graph_id,
+                "content": "The chart shows inplace volumetrics results. ",
+            },
+            {
+                "id": self.table_id,
+                "content": "The table shows statistics per sensitivity. \
+                            Rows can be filtered by searching, and sorted by \
+                            clicking on a column header. ",
```

Comment:
- Using `\` gives strings with a lot of spaces (since the indentation of next line is added as spaces). Visually not a problem I guess here since extra spaces are not rendered in html, but perhaps we should change these multilines to e.g.
    ```python
    "content" = ( "The table shows statistics per sensitivity. "
                        "Rows can be filtered by searching, and sorted by "
                        "clicking on a column header."),
   ```
- Unnecessary space at end of last line.
- Consider replacing _sensitivity_ with _sensitivity parameter_ or _parameter_? (Or keep as is - I'm not sure)